### ISSUE TITLE
v2.1.2 breaks when scale != scale of UIScreen

### ISSUE THREAD

**Issue by PiedPieper** [dehli/TouchDraw#46]
To replicate, simply try this project here: https://github.com/rajeejones/PDFSignature.git and change the Podfile to use 2.1.2 v.s. 2.1.1. In 2.1.2, the bottom 90% of the image is clipped, but if you scale up the touchDrawView to dimensions similar to the iPhone screen, then it will show the whole image on "Add Signature"

**Comment by dehli** [comment]
Thanks @PiedPieper for opening the issue! I'll try to look into it in the next few days. Cheers!

**Comment by wzbozon** [comment]
Yes, `2.1.2` is broken, function `exportDrawing()` doesn't work. Use `2.1.1`.
